tests/Integration/Http/Fixtures/PostResourceWithNoMergeWrap.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Illuminate\Tests\Integration\Http\Fixtures;
+
+use Illuminate\Http\Resources\Json\JsonResource;
+
+class PostResourceWithNoMergeWrap extends JsonResource
+{
+    protected $wrapOnMerge = false;
```

Comment:
Suggestion: Perhaps this should be an option that devs should specify on a per `MergeValue` basis.

```php
   'some_non_scalar_object' => $this->mergeWhen($condition, $this->some_non_scalar_obj)->wrap(false),
   'an_actual_resource' => $this->mergeWhen($condition, $this->related_model),
```